packages/next/src/client/components/react-dev-overlay/_experimental/internal/container/runtime-error/use-error-hook.ts

```diff
@@ -30,8 +30,7 @@ function getErrorSignature(ev: SupportedErrorEvent): string {
       break
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const _ = event satisfies never
+  event satisfies never
```

Comment:
TypeScript already checks `switch` exhaustiveness. No need for this pattern and unused return statements: https://github.com/vercel/next.js/pull/75854